### DISCUSSION TITLE
How to you permanently ban someone?

### DISCUSSION THREAD

**Discussion by 2-bithacker** [andrius4669/zeromod-sauerbraten#15] [Q&A] [answered]
How can I permanently kick someone from my server running the zeromod?  I asked around on discrod,  and nobody knew.

**Comment by andrius4669** [comment, marked as answer]
there's #pban command

**Reply by 2-bithacker** [reply]
thanks!  I recommend making a command reference file that includes all the special commands that this server mod has, so no one will have to ask any questions like this again.

**Reply by andrius4669** [reply]
i tried starting to do that but its too much unfun effort tbh

**Reply by andrius4669** [reply]
https://github.com/andrius4669/zeromod-sauerbraten/blob/master/config.md

**Reply by 2-bithacker** [reply]
Thanks!  one more question.  I heard that the zeromod had a rugby mode.  Is that true?

**Reply by andrius4669** [reply]
sata maintains mod which can do that iirc, it's not in upstream zeromod.
